lib/tic_tac_toe.ex

```diff
@@ -5,26 +5,34 @@ defmodule TicTacToe do
 
   @welcome_message "Welcome to Tic-Tac-Toe"
 
-  @spec start(any(), any()) :: :ok
+  @spec start(any(), any()) :: nil
   def start(communicator, communicator_formatter, board \\ Board.setup_initial_board()) do
     welcome = @welcome_message <> "\n" <> communicator_formatter.format_board(board)
     communicator.display(welcome)
 
+    play_turn(communicator, communicator_formatter, board)
+  end
+
+  @spec play_turn(any(), any(), map()) :: nil
+  defp play_turn(communicator, communicator_formatter, board) do
+    player_turn = Board.calculate_turn(board)
```

Comment:
I'm not sure what I would expect a name like `player_turn` to represent, but I don't think of the markings/symbols of "X" and "O" being "player turns" personally. I wonder if we could come up with another name to represent these.

Reply:
That is a really good point, `player_symbol` sounds much better to me!